dashboard/main.py

```diff
@@ -118,21 +118,32 @@ def build_resample_rate_slider() -> str:
     return f'{resample_rate}T'
 
 
-def build_moisture_header_and_metric(chart_data: pd.DataFrame) -> None:
+def build_moisture_header_and_metric(chart_data: pd.DataFrame, selected_plant: int) -> None:
     """Builds soil moisture header and metric."""
 
+    col1, col2 = st.columns([1, 3])
     latest_data = chart_data.iloc[-1]
-    st.header('🚿 Soil Moisture')
-    st.metric('Current Soil Moisture',
-              f"{latest_data['soil_moisture']:.1f}%", delta='4')
+
+    with col1:
+        st.subheader('🚿 Soil Moisture')
+
+    with col2:
+        st.metric(f'Current Level',
+                  f"{latest_data['soil_moisture']:.1f}%")
 
 
-def build_temperature_header_and_metric(chart_data: pd.DataFrame) -> None:
+def build_temperature_header_and_metric(chart_data: pd.DataFrame, selected_plant: int) -> None:
     """Builds temperature header and metric."""
 
-    latest_data = chart_data.iloc[-1]
-    st.header('🌡️ Temperature')
-    st.metric('Current Temperature', f"{latest_data['temperature']:.1f}°C")
+    col1, col2 = st.columns([1, 3])
+
+    with col1:
+        latest_data = chart_data.iloc[-1]
+        st.subheader('🌡️ Temperature')
+
+    with col2:
+        st.metric(
+            f'Current Temperature (Plant {selected_plant})', f"{latest_data['temperature']:.1f}°C")
 
 
 def display_sidebar_image(selected_plant: int) -> None:
```

Comment:
This look like a huge function, could probably be split up

Reply:
I'm jumping on the file now, so will get working on that